examples/basicPackageCreation.js

```diff
@@ -0,0 +1,33 @@
+const SerialPort = require('serialport')
+const EnoceanParser = require('../ESP3Parser').FullParser
+const Packet = require('../Packet.js')
+const port = new SerialPort('/dev/ttyUSB0', { baudRate: 57600 })
+const parser = port.pipe(new EnoceanParser())
+
+async function doit () {
+  var cc = new Packet.CommonCommand()
+  cc.load({ data: Buffer.from('08', 'hex') }) // Common Command Code 08: CO_RD_IDBASE
+  var res = await sendAndWaitForResponse(cc.getRawBuffer()) // send the telegram asynchronously and wait for response.
+  var idbase = res.data.slice(1, 5).toString('hex') // extract idbase
+
+  // repeat any RadioERP1 Packet but overwrite the id with idbase of this tcm300
+  parser.on('data', data => {
+    if (data.packetTypeNumber === 1) {
+      data.senderId = idbase
+      sendAndWaitForResponse(data.getRawBuffer())
```

Comment:
here is another usecase: repeating all received packets...
As you can only send with your own baseId (+ some offset) you could just overwrite the senderId and resend. 
How would this look like with your API idea?

the real purpose of this is writing complex control logic. Say you have a thermostat that messures the room temperature and allows you to set the temperature. The heating control unit receives the telegram whenever the temperature or the setpoint changes and controlls your valves accordingly.

if you now want to change the setpoint not only in the room, but form different applications like a website. you have to somehow overwrite the setpoint send from the room control unit.

in the end what you have to do is write a middleware that receives all telegrams and repeats them as needed overwriting the setpoint in case you set it via the website.

With enocean, simple things are simple, but as soon as you have a little more complex control needs, you need software. So as soon as you need our module, you will need this feature anyways...